### PR TITLE
#36355 PostgreSQL: Fix default privileges for new roles

### LINKED ISSUE #36355: Default privileges in PostgreSQL for new users aren't displayed in UI
### Description

CE: granted priveleges aren't visible in UI

### DBeaver Version

Version 24.3.0.202411250716

### Operating System

Win11

### Database and driver

PostgreSQL

### Steps to reproduce

1. Open a database, go to roles, create a new one and don't press Savr in the opened dialog
2. Go to Permissions tab, click Tables folder and grant some Permissions
3. Save changes

AR:
CE: granted permissions aren't visible

https://github.com/user-attachments/assets/36eba58e-e3e2-4a6d-a5a5-23242232733b

EE:  granted permissions are visible after dialog reopening

https://github.com/user-attachments/assets/96b30046-b68d-43fb-90d4-552caadc9d1c

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ivgag**
When users edit default privileges for new roles. Roles don't have ids so DBeaver has to use names to find the privileges.

### CODE REVIEW

**Review by ShadelessFox — APPROVED** [review]

**Review by E1izabeth — APPROVED** [review]

**Comment by E1izabeth** [comment]
We need to test it for Redshift also and create corresponding issues, if it doesn't work.

**Comment by ivgag** [comment]
> We need to test it for Redshift also and create corresponding issues, if it doesn't work.

There’s no point in testing it for Redshift as part of this ticket since it has its own implementation: `com.dbeaver.db.redshift.model.RedshiftRole#getPrivileges`.

File: plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreRole.java [hunk 1]
```diff
@@ -492,37 +492,37 @@ public List<PostgrePrivilege> getPrivileges(@NotNull DBRProgressMonitor monitor,
...
+                      "\tn.nspname AS relname,\n" +
+                      "\tcast('C' as \"char\") AS relkind,\n" +
+                      "(aclexplode(nspacl)).grantee as granteeI\n" +
+                      "FROM\n" +
+                      "\tpg_catalog.pg_namespace n\n" +
+                      "WHERE\n" +
+                      "\tn.nspacl IS NOT NULL \n" +
+                      "\t) AS tr\n" +
+                      "WHERE pg_get_userbyid(tr.granteeI)= ? \n" +
+                      " AND tr.relkind IN('S', 'm', 'C')";
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreRole.java]: I believe two more spaces were added unintentionally here. We use 4-space formatting. Could you revert this change?

**Review by E1izabeth — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]
